_observing-your-data/metricsanalytics.md

```diff
@@ -0,0 +1,205 @@
+
+### Visualizing OTel metrics within OpenSearch
+
+To visualize OTel metrics in OpenSearch, follow these steps: 
+
+1. Install the [`opentelemetry-demo` repository](https://github.com/opensearch-project/opentelemetry-demo). See the [Getting Started](https://github.com/opensearch-project/opentelemetry-demo/blob/main/tutorial/GettingStarted.md) guide for instructions.
+2. Collect the OTel signals, including metrics signals. See the [OTel Collector](https://opentelemetry.io/docs/collector/) guide for instructions.  
+3. Configure the OTel pipeline to emit metrics signals. See the [OTel Collector Pipeline](https://github.com/opensearch-project/opentelemetry-demo/tree/main/src/otelcollector) guide for instructions.
+
+#### Example YAML config file
+
+```yaml
+    service:
+      extensions: [basicauth/client]
+      pipelines:
+        traces:
+          receivers: [otlp]
+          processors: [batch]
+          exporters: [otlp, debug, spanmetrics, otlp/traces, opensearch/traces]
+        metrics:
+          receivers: [otlp, spanmetrics]
+          processors: [filter/ottl, transform, batch]
+          exporters: [otlphttp/prometheus, otlp/metrics, debug]
+        logs:
+          receivers: [otlp]
+          processors: [batch]
+          exporters: [otlp/logs,  opensearch/logs, debug]
+```
+{% include copy-curl.html %}
+    
+4. Configure the [Data Prepper pipeline](https://github.com/opensearch-project/opentelemetry-demo/blob/main/src/dataprepper/pipelines.yaml) to emit the collected metrics signals into the OpenSearch metrics index.
+
+#### Example YAML config file
+
+```yaml
+    otel-metrics-pipeline:
+      workers: 8
+      delay: 3000
+      source:
+        otel_metrics_source:
+          health_check_service: true
+          ssl: false
+      buffer:
+        bounded_blocking:
+          buffer_size: 1024 # max number of records the buffer accepts
+          batch_size: 1024 # max number of records the buffer drains after each read
+      processor:
+        - otel_metrics:
+            calculate_histogram_buckets: true
+            calculate_exponential_histogram_buckets: true
+            exponential_histogram_max_allowed_scale: 10
+            flatten_attributes: false
+      sink:
+        - opensearch:
+            hosts: ["https://opensearch-node1:9200"]
+            username: "admin"
+            password: "my_%New%_passW0rd!@#"
+            insecure: true
+            index_type: custom
+            template_file: "templates/ss4o_metrics.json"
+            index: ss4o_metrics-otel-%{yyyy.MM.dd}
+            bulk_size: 4
+```
+{% include copy-curl.html %}
+
+5. Ingest metrics data into OpenSearch. As the demo starts generating data, the metrics signals will be added to the OpenSearch index that supports the OTel Metrics schema format.
```

Comment:
If "OTel Metrics" is a proper noun, then it's most likely "OpenTelemetry Metrics".